test_common/test_zext_wrapper.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import random
+import tempfile
+import fault
+from common.zext_wrapper import ZextWrapper
+
+
+@pytest.mark.parametrize("in_width,out_width", [(5, 10), (10, 5)])
+def test_zext_wrapper(in_width, out_width):
+    if in_width >= out_width:
+        with pytest.raises(ValueError) as pytest_e:
+            ZextWrapper(in_width, out_width)
+            assert False
+        expected_error = ValueError(f"input width must be greater than output "
```

Comment:
Same problem as above. Output width must be > input width

Reply:
Done